features/step_definitions/rubular_steps.rb

```diff
@@ -0,0 +1,17 @@
+require 'watir-webdriver'
+
+Given /^I am on rubular.com$/ do
+  @page= Watir::Browser.new
+  @page.goto "http://rubular.com/"
+end
+
+When /^I enter the regexp and the test string$/ do
+@page.text_field(:id => "regex").set "(.*)@(.*)\\.(.*)"

```

Comment:
Should use page object to define these elements and then call those instance methods.